source/administration/object-management.rst

```diff
@@ -220,6 +220,43 @@ Conversion to or from XML may be required for importing rules created on S3 or s
 
 See :ref:`minio-lifecycle-management` for more complete documentation.
 
+Target Bucket Considerations
+----------------------------
+
+- MinIO does *not* require that the target bucket match object management or versioning configurations with the source bucket.
+- Object locking set on the target bucket may prevent desired ``delete`` operations from the source bucket from completing.
+- If the target bucket has expiration rules configured, these may result in data loss.
+- MinIO tiers objects with their own ``UUID``, so versioning on the target bucket is not required.
+
+  Versioning on the target bucket may have the following unexpected or undesired results:
+  
+  - Excess data usage on the target remote after a delete operation on the source
+  - Reduced storage efficiency on the target, as ``delete`` operations result in ``DeleteMarker``
+  - Duplicate delete markers on source and target buckets
+
+- The target bucket *can* have its own set of object management rules.
+
+  Target buckets should *not* have their own rules for expiration or additional tiering.
```

Comment:
I'd move this up to be in line with 226.

So something like

The remote bucket should not have any ILM expiration or tiering configurations.
Expiration rules can result in removal of tiered data still in use by the source.
Tiering to an additional remote creates an additional network hop between the hot tier and it's data while also increasing operational complexity.

You *may* configure object locking or versioning on the remote bucket. However, doing so has the following consequences:

- The bullet points.